lib/compiler/ast-to-ops.js

```diff
@@ -164,7 +164,10 @@ function compileTableToOps(table, extra_in_params) {
         throw new NotImplementedError(table);
 
     if (table.isVarRef) {
-        return new TableOp.InvokeVarRef(table.name, table.in_params.concat(extra_in_params), table);
+	const compiled = new TableOp.InvokeVarRef(table.name, table.in_params.concat(extra_in_params), table);
+        compiled.device = null;
+        compiled.handle_thingtalk = false;
+        return compiled;
     } else if (table.isResultRef) {
         return new TableOp.ReadResult(table.kind + ':' + table.channel, table.index, table.schema, table);
```

Comment:
This line here has the same problem, can you fix that at the same time?
Also, there is an indent problem.

Reply:
Fixed indent and took a stab at fixing the second instance.